prow/repoowners/repoowners.go

```diff
@@ -490,8 +498,9 @@ func canonicalize(path string) string {
 
 func (o *RepoOwners) walkFunc(path string, info os.FileInfo, err error) error {
 	log := o.log.WithField("path", path)
+	log.Debugf("walkFunc(%q, %q,  %q)", path, info, err)
 	if err != nil {
-		log.WithError(err).Error("Error while walking OWNERS files.")
+		log.WithError(err).Errorf("Error while walking OWNERS files while processing %q", path)
```

Comment:
Another option here would be:
`log.WithError(err).WithField("path", path).Error("Erro while walking OWNERS files")`

Reply:
Ok, reviewing further,  on line 500 log already has a field Entry for path. So I think I can just drop the formatted string altogether.